ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-stable.yaml

```diff
@@ -514,6 +514,9 @@ tests:
   cron: 31 17 22 * *
   steps:
     cluster_profile: ibmcloud-qe-2
+    env:
+      SERVICE_ENDPOINT_COSConfig: DEFAULT_ENDPOINT
```

Comment:
Are these new endpoints supported in 4.16 and 4.17 ?

Reply:
Currently, the PR has not been ported to the previous, plan support 4.15-4.17